### PR TITLE
Video thumbnails

### PR DESCRIPTION
**Pull request by ignapas**
# Description

Added endpoint that looks for ``manifest.xlsx`` files that describe thumbnails for videos in the gallery

## Type of change

Delete those that don't apply.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -737,6 +740,113 @@ def get_osparc_data():
...
+                                    ret[str(file)] = parent_path_without_prefix / row2[0].value
+
+                else:
+                    logging.error("Manifest file in {} lacks at least one required column".format(parent_path))
+                    continue
+
+        try:
+            s3_resp = s3.get_object(
+                Bucket=Config.S3_BUCKET_NAME,
+                Key="{}/{}/files/derivative/pig_heart_Layout1_thumbnail.jpeg".format(102, 5),
```
Comment (dbrnz) [inline app/main.py:831]: Why is the pig heart thumbnail hardcoded here??

**Comment by hsorby** [comment]
This shouldn't be looking for a manifest file in the dataset and consuming it, it should be querying SciCrunch for the information it is looking for.

**Review by Tehsurfer — COMMENTED** [review]
Have you had a look a using scicrunch for accessing manifest.xlsx data? 

That is what we do for accessing manifest.xlsx files in most other cases, although it does have it's own issues.

**Comment by alan-wu** [comment]
As long as the manifest file is curated and indexed by the SciCrunch elastic endpoint - I do not think you need to make any change on the API. You will be able to access the objects level information in the front-end. Changes can be made there instead.

It takes time for SciCrunch to curate the data but it will be easier to wait for that to happen and access the information there instead of looking up to the xlsx file in the bucket.

**Pull request closed without merging**